### PR TITLE
[Fix] fail on unknown attributes

### PR DESCRIPTION
**Pull request by grahamjenson**

### CODE REVIEW

**Comment by heimdall-asguard** [comment]
Reviews      | 0/1
------------ | -------------




[More info](https://heimdall-public.cbhq.net/v1/code_change?repository=coinbase/step&ref=8979f4d5bff1c32248f7d796ba8533a19bbbc435)

File: deployer/release_parsing.go [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+// But the problem is that there are exceptions that we have
+type XReleaseExceptions struct {
+	XRelease
+	Task *string // Do not include the Task because that can be implemented
+}
+
+// UnmarshalJSON should error if there is something unexpected
+func (release *Release) UnmarshalJSON(data []byte) error {
+	var releaseE XReleaseExceptions
```
Comment (chwevans) [inline deployer/release_parsing.go]: can this be named something else?

Comment (grahamjenson) [reply deployer/release_parsing.go]: can be named anything

**Comment by heimdall-asguard** [comment]
Reviews      | 1/1
------------ | -------------
chwevans | 1



[More info](https://heimdall-public.cbhq.net/v1/code_change?repository=coinbase/step&ref=6c83183fe30d556fed322d574ea2f28ef7e91719)

**Comment by heimdall-asguard** [comment]
Reviews      | 1/1
------------ | -------------
chwevans | 1

Pull request is reviewed and can be merged to master!

[More info](https://heimdall-public.cbhq.net/v1/code_change?repository=coinbase/step&ref=5de0347a7a3e9fa4005abc6d3691143cc1f3dbc0)

**Pull request merged**